server/server.js

```diff
@@ -38,9 +40,48 @@ app.get('/projects/:id', (req, res) => {
     });
   });
 });
+// added more routes
+app.post('/projects/:ids', (req, res) => {
+  const query = `INSERT INTO updates (title,description,update_date,comments,likes,project_id,backers_only) VALUES (?,?,?,?,?,${req.params.ids},?)`;
+  const {
+    title, description, update_date, comments, likes, backers_only,
+  } = req.body;
+  db.query(query, title, description, update_date, comments, likes, backers_only, (err, result ) => {
+    if (err) {
+      return res.status(500).send(err);
+    }
+    res.sendStatus(201);
+  });
+});
+
+// delete project
+app.delete('/projects/:ids', (req, res) => {
```

Comment:
Same as before with 'ids' and line 44 and line 69

Reply:
changed it to id